ios/FluentUI.xcodeproj/project.pbxproj

```diff
@@ -250,6 +251,7 @@
 		1168630222E131CF0088B302 /* TabBarItemView.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = TabBarItemView.swift; sourceTree = "<group>"; };
 		1168630322E131CF0088B302 /* TabBarView.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = TabBarView.swift; sourceTree = "<group>"; };
 		118D9847230BBA2300BC0B72 /* TabBarItem.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = TabBarItem.swift; sourceTree = "<group>"; };
+		2A9745DD281733D700E1A1FD /* TableViewTokens.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = TableViewTokens.swift; sourceTree = "<group>"; };
```

Comment:
Did you mean to move the table view tokens? Seems like it should stay un-alphabetical for this review.

Reply:
That is not actually an alphabetization; it looks like this file was merged improperly at some point and the reference to `TableViewTokens` was incorrectly placed in the list of files. This move was automatically done by Xcode to sanitize the project file.